managed/Navigation_civi_remote_settings.mgd.php

```diff
@@ -0,0 +1,28 @@
+<?php
+use CRM_Remotetools_ExtensionUtil as E;
+
+return [
+  [
+    'name' => 'Navigation_civi_remote_settings',
```

Comment:
```suggestion
    'name' => 'Navigation__civi_remote_settings',
```
(two underscores for separating entity type from name, would also apply to the file name)

Reply:
I think it makes sense to stick with the name that `civix export` uses which is `<entity name>_<value of name field>`. So for `Navigation__civi_remote_settings` the value of the name field would need to be changed to `_civi_remote_settings`.